tests/ZendTest/Mvc/Router/PriorityListTest.php

```diff
@@ -137,4 +138,15 @@ public function testPriorityWithNegativesAndNull()
 
         $this->assertEquals(array('bar', 'foo', 'baz'), $orders);
     }
+
+    public function testKey()
```

Comment:
The test should only interarct with a `Chain` instance, not with the internal implementation details of it.

You should simply use `Chain#assemble()` with parameters that show a failure.